CSS/style.css

```diff
@@ -0,0 +1,216 @@
+
+body {
+  font-family: Lato, cocogoose, Tahoma, sans-serif;
+}
+
+header {
+  height: 65px;
+  position: fixed;
+}
+
+.only-desktop {
+  display: none;
+}
+
+.only-mobile {
+  display: block;
+}
+
+.hamburger-icon img {
+  width: 40px;
+  padding-left: 15px;
+  padding-top: 15px;
+}
+
+#headline-section {
+  background: url("../Images/waraMobile.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 100px 12px 20px 15px;
+}
+
+.headline-title-text p {
+  color: #ec5242;
+  font-size: 23px;
+  font-weight: 400;
+  margin: 0;
+}
+
+.headline-title-text h2 {
+  background: -webkit-linear-gradient(#d52b1a, #ec5242);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  font-size: 35px;
+  font-weight: 900;
+  margin: 5px 0;
+}
+
+.headline-box-text {
+  border: 2px solid white;
+  margin-top: 30px;
+}
+
+.headline-box-text p {
+  padding: 0 25px;
+  color: darkslategray;
+  line-height: 23px;
+}
+
+.headline-bottom-text {
+  padding-left: 15px;
+}
+
+.headline-bottom-text h3 {
+  font-size: 25px;
+  font-weight: 900;
+  color: darkslategray;
+  margin-bottom: 0;
+}
+
+.headline-bottom-text p {
+  color: darkslategray;
+}
+
+#main-program {
+  background: #282a30;
+}
+
+.main-program-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
```

Comment:
At Desktop: (768px or wider), Why dont we change the `flex-direction` property to `row` in order to look similar to the design guideline 🤔 

> Design guideline Template
![Screenshot (304)](https://user-images.githubusercontent.com/66112760/141968012-f1c32c2b-1637-46a0-9b8f-39baf3fd2cbe.png)

> Your Design
![Screenshot (305)](https://user-images.githubusercontent.com/66112760/141968126-18f3c551-9d32-4c50-a973-bec6d1a2906f.png)

Reply:
I was trying to finish Day 1 Milestones just the mobile version. I plan to work on the desktop version today as we have been given here...
![image](https://user-images.githubusercontent.com/34578631/141973371-180f905c-c3d2-4c24-a857-8df3e1d19d98.png)

Reply:
Kindly finish all the milestone and project requirements before resubmitting 🙂